code.gs

```diff
@@ -25,15 +25,25 @@ var settings = {
      */
     notificationEmail: 'YOUREMEAILHERE@example.com',
     /*
-     * ID OF THE CONTACTS EVENTS CALENDAR
+     * SOURCE OF THE EVENTS
      *
-     * Open https://calendar.google.com, in the menu on the left click on the arrow next to the
-     * the contacts events calendar (which should have a name like 'Birthdays and events'), choose
-     * 'Calendar settings' and finally look for the "Calendar ID field" (it could be something
-     * similar to the default value of '#contacts@group.v.calendar.google.com', but also really
-     * different from it): copy and paste it between the quotes in the next line.
+     * In your birthday calendar settings you have chosen whether to track the birthdays
+     * of just your Google Contacts or those of your Google Contacts and of your Google
+     * Plus contacts as well.
+     * You can check which setting you chose by opening https://calendar.google.com, clicking
+     * on the arrow next to the the contacts events calendar (which should have a name like
+     * 'Birthdays and events') in the menu on the left and opening 'Calendar settings'.
+     *
+     * You can set this variable to:
+     *  'DEFAULT'              This will make the script follow your Google Calendar setting.
+     *  'CONTACTS_ONLY'        This will force the script to track just the birthdays of your
+     *                         Google Contacts, ignoring your Google Calendar setting.
+     *  'CONTACTS_AND_GPLUS'   This will force the script to track the birthdays of both
+     *                         your Google Contacts and Google Plus contacts, ignoring your
+     *                         Google Calendar setting.
+     *   Anything else will be interpreted as a Google Calendar ID from which to extract the events.
```

Comment:
I suggest changing `will be interpreted as a Google Calendar ID` to `will be interpreted as a hard-coded Google Calendar ID`, just to make it really clear what that sentence means, for people who don't have the context we do.